p4-16/spec/P4-16-spec.mdk

```diff
@@ -8587,11 +8610,13 @@ absence), allowing architecture-independent analysis of P4 programs.
 
 The `deprecated` annotation has a required string argument that is a
 message that will be printed by a compiler when a program is using the
-deprecated construct.  This is mostly useful for annotating library
-constructs, such as externs.
+deprecated construct. This is mostly useful for annotating library
+constructs, such as externs. The parameter needs to be local
```

Comment:
nit: needs to be local -> needs to be a local

Reply:
Would also go with 'must be' over 'needs to be' imo.